editor/src/core/shared/template-path.ts

```diff
@@ -262,18 +269,26 @@ export function filterScenes(paths: TemplatePath[]): StaticInstancePath | Instan
 // FIXME: This should be retired, it's just plain dangerous.
 // Right now this is only used for SpecialNodes (in CanvasMetaData)
 function fromStringUncached(path: string): TemplatePath {
-  const [sceneStr, elementStr] = path.split(SceneSeparator)
-  const scene = sceneStr.split(ElementSeparator).filter((e) => e.length > 0)
+  const elementPathStrs = path.split(SceneSeparator)
+  const [sceneStrs, elementStrs] =
+    elementPathStrs.length > 1
+      ? splitAt(elementPathStrs.length - 1, elementPathStrs)
+      : [elementPathStrs, []]
+  const elementStr = elementStrs[0] ?? null
+  const sceneElementPaths = sceneStrs.map((scene) =>
+    scene.split(ElementSeparator).filter((e) => e.length > 0),
+  )
```

Comment:
I'm totally going to change this to use `Array.pop` because I was an idiot for not doing that in the first place

Reply:
Done